Tools/autotest/arducopter.py

```diff
@@ -1448,6 +1449,67 @@ def fly_motor_fail(self, fail_servo=0, fail_mul=0.0, holdtime=30):
 
         self.do_RTL()
 
+    def fly_motor_vibration(self):
+        """Test flight with motor vibration"""
+        self.context_push()
+        self.set_parameter("INS_LOG_BAT_MASK", 3)
+        self.set_parameter("INS_LOG_BAT_OPT", 2)
+        self.set_parameter("LOG_BITMASK", 45054)
+        self.set_parameter("SIM_VIB_MOT_MAX", 350)
+        self.set_parameter("SIM_GYR_RND", 100)
+        self.set_parameter("SIM_DRIFT_SPEED", 0)
+        self.set_parameter("SIM_DRIFT_TIME", 0)
+        self.reboot_sitl()
+
+        self.takeoff(10, mode="LOITER")
+        self.change_alt(alt_min=50)
+
+        hover_time = 15
+        tstart = self.get_sim_time()
+        self.progress("Hovering for %u seconds" % hover_time)
+        while self.get_sim_time_cached() < tstart + hover_time:
+             attitude = self.mav.recv_match(type='ATTITUDE', blocking=True)
+        tend = self.get_sim_time()
+
+        self.do_RTL()
+        vibe_log = self.buildlogs_path("ArduCopter-vibration-log.bin")
```

Comment:
There is a method to get the currently used binary log file.  Any reason not to use that?

Reply:
👍 I did not know about it 😄

Reply:
@peterbarker I'm probably just being dense but can you point me at it?

Reply:
I tried current_onboard_log_filepath() but it fails, just going to stick with the current approach